ext/mysqli/tests/mysqli_rollback.phpt

```diff
@@ -53,6 +53,14 @@ if (!have_innodb($link))
         echo $exception->getMessage() . "\n";
     }
 
+    $link = my_mysqli_connect($host, $user, $passwd, $db, $port, $socket);
+    mysqli_report(MYSQLI_REPORT_ERROR|MYSQLI_REPORT_STRICT);
```

Comment:
```suggestion
```

should be not needed because of https://wiki.php.net/rfc/mysqli_default_errmode

Reply:
That's what I thought at first, but it seems like this is a global setting. If I didn't set it like this here, I ended up in warning mode.

Reply:
Warning mode is set at the beginning of this test case, and if I do not change the mode with this line, the warning mode will be inherited even if I create a new mysqli instance.

Reply:
I see - https://github.com/php/php-src/blob/832c30f38df10fdab235894fa395004dd983d660/ext/mysqli/tests/connect.inc#L10 - `new mysqli_driver` is an instance of the monostate pattern - https://www.php.net/manual/en/class.mysqli-driver.php

Reply:
right.